### DISCUSSION TITLE
Is there a way to search in a specific folder?

### DISCUSSION THREAD

**Discussion by nibreh** [datenstrom/community#253] [Ask a question] [answered]
Hi.

Is there a way to search in a specific folder, and not on the entire site with [search plugin](https://github.com/datenstrom/yellow-extensions/tree/master/source/search) ?

**Comment by BsNoSi** [comment]
See description of plugin:

```
The following settings can be configured in file system/config/config.ini:

SearchLocation = search location
SearchPaginationLimit = number of entries to show per page
SearchPageLength = maximum page length to show
```

Simply set `search location` as desired.

**Comment by markseuffert** [comment]
I am sorry, this is currently not possible. The search plugin searches through the entire website.

To give you an alternative, you can make a custom plugin and then tell it to search a specific location. Replace `$this->yellow->pages->index($showInvisible, false)` with `$this->yellow->pages->find("/location/")->getChildren($showInvisible)`. For more information see [API](https://datenstrom.se/yellow/help/api-for-developers).

Hope this helps. :smile:

**Comment by schulle4u** [comment, marked as answer]
The [search extension](https://github.com/annaesvensson/yellow-search) now supports filtering results by folder. Example: searching for `coffee folder:help` will return only pages containing the term "coffee" within the "help" content folder. All available filters can be found in the documentation.

**Reply by markseuffert** [reply]
I edited the link to the search extension's new repository. Hope that's okay with you.
